src/components/ProfileValidation/ReturnToAppButton.tsx

```diff
@@ -1,27 +1,106 @@
+import {
```

Comment:
this is not a button that opens a confirmation modal rather than just redirecting